sdk/python/kfp/compiler/test_data/components/unsupported/output_named_tuple.py

```diff
@@ -39,10 +39,6 @@ def output_named_tuple(
         }]
     })
 
-    with open(artifact.path) as f:
-        artifact_contents = f.read()
-    model = 'Model contents: ' + artifact_contents
-
     from collections import namedtuple
     output = namedtuple('Outputs', ['scalar', 'metrics', 'model'])
     return output(scalar, metrics, model)
```

Comment:
I think the `model` variable is not defined here

Reply:
Somehow missed this comment. Fixed.